.github/workflows/on-main-push.yaml

```
@@ -50,9 +50,9 @@ jobs:
       - name: Create package distribution and publish
         if: steps.check-version.outputs.NEW_VERSION == 'true'
         run: |
-          cd packages/${{ matrix.package }}
           yarn install
-          yarn build
-          yarn publish --access public
+          yarn build:packages
+          yarn workspace dm-core publish --access public
+          yarn workspace dm-core-plugins publish --access public
```

I thought it were
```suggestion
          yarn workspace @development-framework/dm-core publish --access public
          yarn workspace @development-framework/dm-core-plugins publish --access public
```
? Don't really know this syntax well though

---

I think you're right yeah! Thanks 👍🏻 Hard to test without committing and seeing how it runs 😅